plugins/obs-webrtc/whip-output.cpp

```diff
@@ -5,12 +5,12 @@ const char signaling_media_id_valid_char[] = "0123456789"
 					     "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
 					     "abcdefghijklmnopqrstuvwxyz";
 
-const uint32_t audio_ssrc = 5002;
+const uint32_t audio_ssrc = generate_random_u32();
 const char *audio_mid = "0";
 const uint32_t audio_clockrate = 48000;
 const uint8_t audio_payload_type = 111;
 
-const uint32_t video_ssrc = 5000;
+const uint32_t video_ssrc = generate_random_u32();
```

Comment:
we need to make sure that `video_ssrc` is not the same as `audio_ssrc`

Reply:
Its not

Reply:
there is no warranty that the values returned by the function in two consecutive invocations can't be equal. The probability is low (very low), but nevertheless possible.

Reply:
If we have multiple output from obs-webrtc, eg. stop and start again or start multiple outputs by a future change, the same SSRCs will be used among multiple outputs. Is that ok?
If not, `video_ssrc` and `audio_ssrc` should be generated for each instance and stored in the class `WHIPOutput`.